internal/provider/resource_cloudflare_fallback_domain.go

```diff
@@ -53,16 +81,28 @@ func resourceCloudflareFallbackDomainUpdate(ctx context.Context, d *schema.Resou
 		return diag.FromErr(fmt.Errorf("error setting domain attribute: %w", err))
 	}
 
-	d.SetId(accountID)
+	d.SetId(fmt.Sprintf("%s/%s", accountID, policyID))
```

Comment:
Could you elaborate? I don't believe this is a breaking change. I've tried testing this PR on an existing configuration and no state migrations were needed. We will automatically migrate the old ID format (account tag) in an existing installation to an ID of the new format (accountTag/UUID or default) when the fallback domain/split tunnel is updated

Reply:
sure, my concern here is that we are changing the ID after the resource has been created and if anyone has referenced that ID in another attribute or outputted it, this will break that relationship. generally, in Terraform, IDs are assumed immutable for resources after creation.

i don't have insights into how common that would be with this however, it's definitely possible hence why if i was asking if we have an alternative, i would much prefer to take that route 😃

Reply:
Gotcha, it would be very common for ID to change. All existing `split_tunnel`s and `fallback_domain`s will have their IDs changed from `<accountTag>` to `<accountTag>/default` whenever they're next changed/applied.
My plan: get rid of the `/default` suffix from the ID in this PR.
- If `policyID` is absent from the configuration (which it is for all current existing and "default" `split_tunnel`/`fallback_domain`) then leave the ID as the account tag as it is now.
- If `policyID` is present, then make the ID `<accountID>/<policyID>`